gradle-mvn-push.gradle

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (C) Chris Banes
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This script is derived from the version here:
+// https://github.com/uber/android-template/blob/master/gradle/gradle-mvn-push.gradle
```

Comment:
Ping on this comment.

Since this file is apparently copied from somewhere, could you add a comment at the very top of this file indicating where you copied it from?

Reply:
I removed this file - with Maven-publish this won't be necessary at all. For your information, it comes from the object construction checker repository: https://github.com/kelloggm/object-construction-checker/blob/master/gradle-mvn-push.gradle, which should also probably upgrade.